### PR TITLE
Update user's last_login field in create_token, refresh_token mutations

### LINKED ISSUE #5037: When user login from GraphQL, last_login field is not updated
### What I'm trying to achieve
I would like to be able to know when an user last logged-in from the GraphQL API (JWT).

### Steps to reproduce the problem
1. Look at `last_login` in the user model, remember the value
1. Login using JWT
1.. Look at `last_login` in the user model, it shouldn't have been updated

### What I expected to happen
- `last_login` to be updated when `createToken`
- `last_login` to be updated when `refreshToken`

### PR DESCRIPTION
**Pull request by kswiatek92**
closes #5037 

# Impact

* [ ] New migrations
* [ ] New/Updated API fields or mutations
* [ ] Deprecated API fields or mutations
* [ ] Removed API types, fields, or mutations

# Pull Request Checklist

* [x] Privileged queries and mutations are guarded by proper permission checks
* [x] Database queries are optimized and the number of queries is constant
* [x] Database migration files are up to date
* [x] The changes are tested
* [x] GraphQL schema and type definitions are up to date
* [ ] Changes are mentioned in the changelog

### CODE REVIEW

File: saleor/graphql/core/mutations.py [hunk 1]
```diff
@@ -641,6 +645,21 @@ def resolve(cls, root, info, **kwargs):
...
 
+class RefreshToken(Refresh):
+    """Mutation that refresh user token.
+
+    It overrides the default graphql_jwt.Refresh to update user's last_login field.
+    """
+
+    @classmethod
+    def mutate(cls, root, info, **kwargs):
+        result = super().mutate(root, info, **kwargs)
```
Comment (NyanKiyoshi) [inline saleor/graphql/core/mutations.py:656]: What does it return?

Comment (maarcingebala) [reply saleor/graphql/core/mutations.py:656]: I guess
```
type Refresh {
    token: String
    payload: GenericScalar
}
```

Comment (kswiatek92) [reply saleor/graphql/core/mutations.py:656]: yes

File: tests/api/test_account.py [hunk 2]
```diff
@@ -89,10 +90,14 @@ def test_create_token_mutation(api_client, staff_user, settings):
...
     """
     variables = {"email": staff_user.email, "password": "password"}
-    response = api_client.post_graphql(query, variables)
+    time = timezone.now()
+    with freeze_time(time):
+        response = api_client.post_graphql(query, variables)
     content = get_graphql_content(response)
     token_data = content["data"]["tokenCreate"]
     token = jwt.decode(token_data["token"], settings.SECRET_KEY)
+    staff_user.refresh_from_db()
```
Comment (NyanKiyoshi) [inline tests/api/test_account.py:99]:
```suggestion
    staff_user.refresh_from_db(update_fields=["last_login"])
```

**Pull request merged**